contracts/examples/crypto-kitties/kitty-genetic-alg/src/lib.rs

```diff
@@ -0,0 +1,45 @@
+#![no_std]
+
+imports!();
+
+use kitty::{kitty_genes::*, Kitty};
+use random::Random;
+
+#[elrond_wasm_derive::contract(KittyGeneticAlgImpl)]
+pub trait KittyGeneticAlg {
+	#[init]
+	fn init(&self) {}
+
+	// endpoints
+
+	#[endpoint(generateKittyGenes)]
+	fn generate_kitty_genes(&self, matron: Kitty, sire: Kitty) -> SCResult<KittyGenes> {
+		let mut random = Random::new(*self.get_block_random_seed());
```

Comment:
Isn't the block random seed random enough?
Oh, I see, we need more than 1 number.
If we just need 2 random numbers 1-99, I think we can split the block random seed. (Thinking now, if we shouldn't provide something from the framework that does that).

Reply:
In the current implementation, yes, I could simply take the block random seed and split it, but I prefer having it this way for easier extension later.